cmd/zrepl/mgmt_conn.c

```diff
@@ -557,6 +558,27 @@ uzfs_zvol_rebuild_status(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
 	return (reply_data(conn, &hdr, &status_ack, sizeof (status_ack)));
 }
 
+static int
+uzfs_zvol_stats(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
+    const char *name, zvol_info_t *zinfo)
```

Comment:
just a minor nit:
'name' doesn't seem to be in use. It can be removed.

Reply:
thanks, I will fix that